media-video/motion/motion-4.3.1.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools readme.gentoo-r1 systemd
+
+DESCRIPTION="A software motion detector"
+HOMEPAGE="https://motion-project.github.io"
+SRC_URI="https://github.com/Motion-Project/${PN}/archive/release-${PV}.tar.gz -> ${PN}-release-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="ffmpeg mariadb mmal mysql postgres sqlite supervise-daemon v4l webp"
+
+RDEPEND="
+	acct-group/motion
+	acct-user/motion
+	virtual/jpeg:=
+	ffmpeg? ( media-video/ffmpeg:0= )
+	mariadb? ( dev-db/mariadb-connector-c )
+	mmal? ( media-libs/raspberrypi-userland )
+	mysql? ( dev-db/mysql-connector-c )
+	postgres? ( dev-db/postgresql:= )
+	sqlite? ( dev-db/sqlite:3 )
+	webp? ( media-libs/libwebp:= )
+"
+DEPEND="${RDEPEND}
+	net-libs/libmicrohttpd
+	v4l? (
+		media-libs/libv4l
+		virtual/os-headers
+	)
+"
+
+DISABLE_AUTOFORMATTING="yes"
+DOC_CONTENTS="
+You need to setup a configuraton file (/etc/motion/motion.conf) before
+running motion for the first time.
+
+If motion is built with the mysql or mariadb use flags then please make
+sure to configure a matching database_type in the config file.
+
+Motion runs by default under user motion and group motion:
+- change this if needed in /etc/conf.d/motion
+- or add users who need access to the output files to the motion group
+
+To install motion as a service, use:
+- rc-update add motion default # with OpenRC
+- systemctl enable motion.service # with systemd
+"
+
+S="${WORKDIR}"/${PN}-release-${PV}
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_with ffmpeg) \
+		$(use_with mariadb) \
+		$(use_with mmal) \
+		$(use_with mysql) \
+		$(use_with postgres pgsql) \
+		$(use_with sqlite sqlite3) \
+		$(use_with v4l v4l2) \
+		$(use_with webp) \
+		--without-optimizecpu
+}
+
+src_install() {
+	emake \
+		DESTDIR="${D}" \
+		docdir=/usr/share/doc/${PF} \
+		examplesdir=/usr/share/doc/${PF}/examples \
+		install
+
+	# build INITDAD string cwto be added to initd based on use flags, eg
+	# "depend() {
+	#   use mysql
+	# }
+	#
+	# supervisor=supervise-daemon"
+	local INITDADJ=""
```

Comment:
Also while at it, please make a local variable smallcase. We distinct them from global variables this way. Global variables cannot be changed inside a phase while local can.

Reply:
It is weird that is shows installing in /usr/local on your machine. I see this:

$ equery f motion
 * Searching for motion ...
 * Contents of media-video/motion-4.3.1:
/etc
/etc/conf.d
/etc/conf.d/motion
/etc/init.d
/etc/init.d/motion
/etc/motion
/etc/motion/camera1-dist.conf
/etc/motion/camera2-dist.conf
/etc/motion/camera3-dist.conf
/etc/motion/camera4-dist.conf
/etc/motion/motion-dist.conf
/lib
/lib/systemd
/lib/systemd/system
/lib/systemd/system/motion.service
/usr
/usr/bin
/usr/bin/motion
/usr/share
/usr/share/doc
/usr/share/doc/motion-4.3.1
/usr/share/doc/motion-4.3.1/COPYING.bz2
/usr/share/doc/motion-4.3.1/CREDITS.bz2
/usr/share/doc/motion-4.3.1/README.gentoo.bz2
/usr/share/doc/motion-4.3.1/mask1.png
/usr/share/doc/motion-4.3.1/motion_build.html
/usr/share/doc/motion-4.3.1/motion_config.html
/usr/share/doc/motion-4.3.1/motion_guide.html
/usr/share/doc/motion-4.3.1/motion_stylesheet.css
/usr/share/doc/motion-4.3.1/normal.jpg
/usr/share/doc/motion-4.3.1/outputmotion1.jpg
/usr/share/doc/motion-4.3.1/outputnormal1.jpg
/usr/share/locale
/usr/share/locale/de
/usr/share/locale/de/LC_MESSAGES
/usr/share/locale/de/LC_MESSAGES/motion.mo
/usr/share/locale/nl
/usr/share/locale/nl/LC_MESSAGES
/usr/share/locale/nl/LC_MESSAGES/motion.mo
/usr/share/man
/usr/share/man/man1
/usr/share/man/man1/motion.1.bz2

Reply:
Happens with `FEATURES="test"`.